main.go

```diff
@@ -65,15 +66,6 @@ func main() {
 		newLeader = leader
```

Comment:
It's not temporary as it's defined above on line 56. I suppose the comment left there is trying to explain the why.